frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/secret.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { SecretData } from '@console/internal/components/configmap-and-secret-data';
+import { k8sGet } from '@console/internal/module/k8s/resource';
+import { SecretModel, ConfigMapModel } from '@console/internal/models';
+import { Base64 } from 'js-base64';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { getName, getNamespace } from '@console/shared/src/selectors/common';
+import { isBound } from '../../utils';
+
+export const GetSecret: React.SFC<GetSecretProps> = ({ obj }) => {
+  const secretData = {
+    BUCKET_NAME: Base64.encode(_.get(obj, 'metadata.name')),
+    ACCESS_KEY: Base64.encode('loading..'),
+    SECRET_KEY: Base64.encode('loading..'),
+    ENDPOINT: Base64.encode('http://loading.com'),
+  };
+  if (isBound(obj)) {
+    const secret = k8sGet(SecretModel, getName(obj), getNamespace(obj));
+    const configMap = k8sGet(ConfigMapModel, getName(obj), getNamespace(obj));
+    Promise.all([secret, configMap])
+      .then((datas) => {
+        secretData.ACCESS_KEY = Base64.encode(_.get(datas[0], 'data.AWS_ACCESS_KEY_ID'));
```

Comment:
data is plural only. I don't think there is any word as `datas`

Reply:
ack